mrp_flattened_bom_xlsx/readme/CONTRIBUTORS.rst

```diff
@@ -1,2 +1,3 @@
-* Héctor Villarreal <hector.villarreal@forgeflow.com>
+* Héctor villarreal <hector.villarreal@forgeflow.com>
```

Comment:
I hate to be this nitpicking but... last names start in capital :)

Reply:
Done! :+1: